### ISSUE TITLE
Mismatch of agnpy's absorption with Finke (2016) reference: DT

### ISSUE THREAD

**Issue by cosimoNigro** [cosimoNigro/agnpy#66]
As suggested by @jsitarek in issue #50, I am opening a different issue for each of the absorption crosschecks.
This one regards the absorption on the photon field of the DT.
I obtained the opacity vs energy computed at several distances by Finke - before I was using values I had fetched with webplotdigitizer from the paper's figures.

<img src="https://user-images.githubusercontent.com/26409711/103149971-230e1700-476f-11eb-92cc-0c4d21196e6b.png" width=70%>
<img src="https://user-images.githubusercontent.com/26409711/103149970-230e1700-476f-11eb-9e67-8ea58723d5e5.png" width=70%>
<img src="https://user-images.githubusercontent.com/26409711/103149967-22758080-476f-11eb-9a9e-becc6eab4ce3.png" width=70%>
<img src="https://user-images.githubusercontent.com/26409711/103149966-21dcea00-476f-11eb-9b93-337ae0e495f6.png" width=70%>

Here again the same problem of the BLR in issue #65 repeats: for small distances, within the disk, there is agreement, at larger distances `r=10^2 R(Ly alpha)` there is mismatch. 
On the other hand, at very large distances we have the consistency check from the approximation with the point-source behind the jet.
<img src="https://user-images.githubusercontent.com/26409711/103150098-cd3a6e80-4770-11eb-870f-8bf89433e799.png" width=70%>
So I am a bit confused.

**Comment by jsitarek** [comment]
the agreement looks very nice, closing

**Comment by jsitarek** [comment]
sorry for reopening it, but I found something strange in DT absorption while investigating the BLR one.
![finke_dt_1 03](https://user-images.githubusercontent.com/33022433/108228583-489c1900-713f-11eb-85f1-665d0a8a0930.png)
![finke_dt_10](https://user-images.githubusercontent.com/33022433/108228584-4934af80-713f-11eb-8c63-6c89d72470b7.png)
![finke_dt_100](https://user-images.githubusercontent.com/33022433/108228580-48038280-713f-11eb-9e3e-2173c8cbbfd5.png)

if r << R_dt the reference line and the agnpy line are basicaly shifted by a factor very close to 1+z. This is something that we were discussing before, but from the paper it seems that the curves are given in energy at the galaxy, and actually for BLR absorption from the same plot there is a very nice agreeement. 

if the emission region is far there starts to be a disagreement between agnpy and Finke's paper. 
However I'm not sure of the actual parameters, table3 gives parameters for extended dust torus, while the plot is for point like. I used T=1000K (which is probably fine), and R_dt = 3.5e19 cm (which is the outer DT radius in the table for the extended DT).
@cosimoNigro which parameters have you used in your comparison plots?

**Comment by jsitarek** [comment]
Following the discussion that we had today with @cosimoNigro I updated the plots with the proper R_dt= 1.6e19 cm (using formula 96 from the Finke'16 paper). If the Finke's points are scaled up by a factor of 2 (an issue in that paper that we discussed earlier), and are corrected by redshift of then I get a perfect agreement like Cosimo in his plots. We can close again the issue. However I think it is worth to write to Justin Finke that the BLR and DT absorption points in his Fig 14 seem to be computed w.r.t. different energies (BLR in the source frame, DT in the observer frame).
